planet/src/main/webapp/js/placesItinerary.js

```diff
@@ -38,10 +38,18 @@ function initAutocomplete() {
     let options = {
         types: ['geocode']
     };
-    autocompleteStart = new google.maps.places.Autocomplete(startAddress,options); 
+    autocompleteStart = new google.maps.places.Autocomplete(startAddress,options);
+    autocompleteStart.setFields(['address_component']);
+    autocompleteStart.addListener('place_changed', updateStartingAddress);
+
     autocompleteEvent = new google.maps.places.Autocomplete(eventAddress,options);
 }
 
+function updateStartingAddress() {
+    const startAddress = document.getElementById('starting-address');
+    sessionStorage.setItem('start', startAddress.value);
```

Comment:
is this related to this PR? It looks like a separate fix

Reply:
I talked about it in the PR description (since all 3 fixes are relatively small I combined them into one PR)